app/Listeners/VisitTransfer/NotifyAllReferees.php

```diff
@@ -17,22 +21,34 @@ public function __construct()
     public function handle(ApplicationSubmitted $event)
     {
         $refereesToBeNotified = $event->application->referees->filter(function ($ref) {
-            return !$ref->is_requested && !$ref->is_submitted;
+            if($ref->reminded_at == null){
+                return !$ref->is_submitted;
+            }
+            return $ref->reminded_at->lte(Carbon::now()) && !$ref->is_submitted;
```

Comment:
As far as I can tell, this listener should be fired once and only once, after an application has been submitted - if so, is there ever a situation where `reminded_at` would be set at this point?

Reply:
This listener is set later to be called again for the second time for the follow up emails, so references that are not submitted will have reminded at set, which would cause an error in the following line